src/host/ft_host/CJK_DbcsTests.cpp

```diff
@@ -2038,3 +2042,44 @@ void DbcsTests::TestDbcsStdCoutScenario()
     VERIFY_ARE_EQUAL(cchReadBack, dwRead, L"We should have read as many characters as we expected (length of original printed line.)");
     VERIFY_ARE_EQUAL(String(test), String(psReadBack.get()), L"String should match what we wrote.");
 }
+
+// Read/WriteConsoleOutput allow a user to implement a restricted form of buffer "backup" and "restore".
+// But what if the saved region clips ("bisects") a wide character? This test ensures that we restore proper
+// wide characters when given an unpaired trailing/leading CHAR_INFO in the first/last column of the given region.

```

Comment:
for my education . . .

does this mean that if you send `{u+202e, trailing}`, it will move one cell to the left and emit a leading copy of it?

can this be documented graphically in the test?